src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -314,6 +315,8 @@ namespace SurfaceGeometry {
             return;
         }
 
+        ZoneEquipmentManager::GetZoneEquipment(state); // Necessary to get this before window air gap code
```

Comment:
I haven't looked yet, but I assume this is causing CI diffs just due to reordering, right?